ambari-server/src/main/java/org/apache/ambari/server/orm/DBAccessorImpl.java

```diff
@@ -1574,7 +1574,7 @@ protected String buildQuery(String tableName, String[] requestedColumnNames, Str
     }
     builder.append(requestedColumnNames[0]);
     for (int i = 1; i < requestedColumnNames.length; i++) {
-      builder.append(", ").append(requestedColumnNames[1]);
+      builder.append(", ").append(requestedColumnNames[i]);
```

Comment:
Can you please add a third column in the corresponding unit test to catch such typo?

https://github.com/apache/ambari/blob/e62cc462fb5d2ac0f2c90912d044db352c477c74/ambari-server/src/test/java/org/apache/ambari/server/orm/DBAccessorImplTest.java#L833-L834

```diff
-    assertEquals(String.format("SELECT id, name FROM %s", tableName),
-    dbAccessor.buildQuery(tableName, new String[] {"id", "name"}, null, null));
+    assertEquals(String.format("SELECT id, name, time FROM %s", tableName),
+    dbAccessor.buildQuery(tableName, new String[] {"id", "name", "time"}, null, null));
```

```
org.junit.ComparisonFailure: expected:<SELECT id, name, [ti]me FROM test_table_3...> but was:<SELECT id, name, [na]me FROM test_table_3...>
	at org.apache.ambari.server.orm.DBAccessorImplTest.testBuildQuery(DBAccessorImplTest.java:833)
```

Reply:
Thank you for suggestion! Added